docs/user-guide/kubectl-overview.md

```diff
@@ -238,6 +238,12 @@ $ kubectl get replicationcontroller <rc-name>
 
 // List all replication controllers and services together in plain-text output format.
 $ kubectl get rc,services
+
+// List all daemon sets, including uninitialized ones, in plain-text output format.
+$ kubectl get ds --include-uninitialized
+
+// List all daemon sets, not including uninitialized ones, in plain-text output format.
+$ kubectl get ds --include-uninitialized=false
 ```
 
 `kubectl describe` - Display detailed state of one or more resources.
```

Comment:
`kubectl describe` includes the uninitialized objects, right? Could you add that to the comment?

Reply:
+1